src/judging_round_selector.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import { Form, Select } from 'semantic-ui-react'
+
+
+class JudgingRoundSelector extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
```

Comment:
Same comment here about being able to remove some boiler plate. This could be:

```js
class JudgingRoundSelector extends React.Component {
  state = {
    error: null,
    isLoaded: false,
    user: []
  }

Reply:
Done.